crates/ra_hir_expand/src/builtin_macro.rs

```diff
@@ -208,23 +245,44 @@ fn format_args_expand(
     Ok(expanded)
 }
 
+fn unquote_str(lit: &tt::Literal) -> Option<String> {
+    let lit = ast::make::tokens::literal(&lit.to_string());
+    let token = ast::String::cast(lit)?;
```

Comment:
Long term, we probably should have a way to uquote a `tt` without going via AST, but this totally fine for the time being

Reply:
When I implement this functions, I tried to find whether we have a util functions for unquoting string and I found this and other one inside `ra_syntax` (`validate_literal`). I would be happy to implement one but I don't know where should it in, Maybe `ra_fmt` ? Anyway it will be another PR.